README.md

```diff
@@ -552,6 +552,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :bulb: [Using multiple Git accounts with SSH keys](https://gist.github.com/oanhnn/80a89405ab9023894df7)
 - :green_book: [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)
 - :video_camera: [Kunal Kushwaha's Complete Git and GitHub tutorial](https://www.youtube.com/watch?v=apGV9Kg7ics)
+- :video_camera: [Learn Git and GitHub in the best way!!](https://www.youtube.com/watch?v=gwWKnnCMQ5c&t=2829s)
```

Comment:
Love the enthusiasm for the title, but can you change it to the title of the video?

Reply:
@kartikver15gr8 please make this change